### PR TITLE
Run PHP 7.4 tests in parallel with existing 7.2

### PR DESCRIPTION
**Pull request by MKodde**
Separate Dockerfiles where created for the php fpm container. This
trickled down to a similar docker-compose and init script.

The test integration github actions file was updated to start both jobs

### CODE REVIEW

**Comment by MKodde** [comment]
I have not found a Github Action instruction to mark a job as 'allowed to fail'. Is this an issue? Or do you see other options here @pablothedude ?

File: ci/docker/docker-compose-php74.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+version: '2.2'
+
+services:
```
Comment (pablothedude) [inline ci/docker/docker-compose-php74.yml:3]: I would suggest only add config to overide. So it's clear what is overridden. FYI: you can validate the end result with docker config

File: ci/docker/init-php74.sh [hunk 2]
```diff
@@ -4,9 +4,9 @@ gid=$(id -g)
 
 printf "UID=${uid}\nGID=${gid}\nCOMPOSE_PROJECT_NAME=gateway" > .env
 
-docker-compose up -d --build
+docker-compose -f docker-compose-php74.yml up -d --build
```
Comment (pablothedude) [inline ci/docker/init-php74.sh:7]: Maybe this should be moved to the Github workspace config.

File: ci/docker/init-php74.sh [hunk 3]
```diff
@@ -4,9 +4,9 @@ gid=$(id -g)
...
 
-docker-compose exec -T php-fpm.stepup.example.com bash -c '
+docker-compose -f docker-compose-php74.yml exec -T php-fpm.stepup.example.com bash -c '
```
Comment (pablothedude) [inline ci/docker/init-php74.sh:9]: Maybe this should be removed to the Dockerfile.

**Comment by MKodde** [comment]
Yes, I will make note of your feedback, and let's address it in the near future

**Pull request merged**